### PR TITLE
Initial build requirements

### LINKED ISSUE #1: Initial Build
**Background**
The existing Node package (below) looks fairly robust so this should be a relatively easy build. We're only building out the Core API for now.

**Requirements**
- Build methods for `list` from the following endpoints:
  - [ ] [Activity Stream](https://developer.zendesk.com/rest_api/docs/core/activity_stream)
  - [ ] [Tickets](https://developer.zendesk.com/rest_api/docs/core/tickets)
  - [ ] [Ticket Metrics](https://developer.zendesk.com/rest_api/docs/core/ticket_metrics)
  - [ ] [Ticket Forms](https://developer.zendesk.com/rest_api/docs/core/ticket_forms)
  - [ ] [Users](https://developer.zendesk.com/rest_api/docs/core/users)
  - [ ] [Groups](https://developer.zendesk.com/rest_api/docs/core/groups)
  - [ ] [Group Memberships](https://developer.zendesk.com/rest_api/docs/core/group_memberships)
  - [ ] [Tags](https://developer.zendesk.com/rest_api/docs/core/tags)
  - [ ] [Schedules](https://developer.zendesk.com/rest_api/docs/core/schedules)
  - [ ] [Satisfaction Ratings](https://developer.zendesk.com/rest_api/docs/core/satisfaction_ratings)
  - [ ] [Sessions](https://developer.zendesk.com/rest_api/docs/core/sessions)
  - [ ] [Brands](https://developer.zendesk.com/rest_api/docs/core/brands)
  - [ ] [Support Addresses](https://developer.zendesk.com/rest_api/docs/core/support_addresses)
  - [ ] [Twitter](https://developer.zendesk.com/rest_api/docs/core/twitter_channel)

**Relevant Links**
https://developer.zendesk.com/rest_api/docs/api-clients/nodejs
https://www.npmjs.com/package/node-zendesk

### PR DESCRIPTION
**Pull request by noahbass**
A few methods are not covered by node-zendesk and are instead covered by the `apiGetRequest method`: `listTicketForms`, `listSchedules`, `listBrands`, `listSupportAddresses`, and `listTwitter`

Closes #1

### CODE REVIEW

**Comment by trong-le** [comment]
@noahbass Could you add descriptions to what each method returns along with what an example token would look like. [Examples for both can be found here](https://github.com/aries-data/aries-activity-github-source)

**Comment by noahbass** [comment]
Updated to include descriptions and an example token @trong-le

File: README.md [hunk 1]
```diff
@@ -73,7 +75,7 @@ Tokens are created and destroyed in your organization's Zendesk support admin in
 Example call - getting a list of tickets
 
 ```javascript
-const source = new ZendeskSource('you@example.com', 'token', 'example'); // email, token, subdomain
+const source = new ZendeskSource('noah@astronomer.io', 'jeQaMUaF80YuYg7j5uwPhzb4jSx76h5bmDtnoKXT', 'astronomer'); // email, token, subdomain
```
Comment (trong-le) [inline README.md]: @noahbass is this a real access token or just an example one?

Comment (noahbass) [inline README.md]: It's an example token 😃  @trong-le

**Pull request merged**